foo.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func GetAPIKey(keyName string) string {
+	// loads values from .env into the system
+	if err := godotenv.Load(); err != nil {
+		log.Fatal(err)
+	}
+
+	// gets API_KEY
+	apiKey, exists := os.LookupEnv(keyName)
+	if exists {
+		return apiKey
+	}
+	return ""
+}
+
+// implement using ordered maps https://github.com/wk8/go-ordered-map/blob/v2.1.7/orderedmap.go#L23
+func newFields(fields ...string) {
+	return
+}
+
+func buildURL(base, endpoint string, fields map[string]string) {
+
+}
```

Comment:
if these functions are not used, we'd rather remove them

Reply:
Removed them in further commits. Those were meant to generalize API consumption.

Reply:
I don't get it. why are they still here?

Reply:
I renamed the file `utils.go`
did `git add utils.go` then pushed it
It created a new file in the repo (didn't overwrite `mempool.go`)

Check the file `utils.go` for changes. I''l remove the other one.